team13-fydp/fydp
# Simple Master Output

This branch is to print the list of teacher, cohort, subject, day and period assignments for the entire school. This will be printed as a list to the already existing sheet called Output Sheet.

To Test:
1. Change the file path to be "flamborough_feb6.xlsx" as this file contains the input information
2. Open the file and make sure the output sheet is cleared. Make sure you close the Excel file before running 
3. Run the model. When it finishes, open the flamborough_feb6.xlsx file to ensure it was correctly printed over
4. Check that the correct teacher names are printing in Excel to match what is outputted in the Java Console (ex. if Java says Teacher 0, the teacher name printing is the first one entered etc.)
5. Check that the correct cohort is being printed (same process as teacher)
6. Check subject, day and period matches as well. Do this with a couple random rows to make sure everything matches

Note: Every time this code is run, it will overwrite what was existing in the sheet (**does not** make a new sheet with timestamp each time)
